### DISCUSSION TITLE
Freeplane 1.11.13 released

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#1810] [Announcements]
Freeplane 1.11.13 contains latest filter and code explorer improvements back ported from 1.12.x. 
I have also recorded some Code Explorer videos. Look at them. I think I could also record some videos about Freeplane itself in this way, if I get encouraging feedback.

https://youtu.be/GvA3i_MBC-4
https://youtu.be/NvNZUydrCOg
https://youtu.be/8_v5nm7caWk

**Comment by macmarrum** [comment]
That's awesome!

Thanks for shring the videos :heart: 

It's great to see Code Explorer in action. It also shows the power of Freeplane as a versatile mind mapping tool.

I know that people can use Freeplane in different ways. I think that videos where people show what they do often, i.e. for which they've become "experts" or what gets them going, what fascinates them, are the most attractive to watch. I would be interesetd in such videos, be it about the usage of Code Explorer or about other things that can be done in Freeplane :+1:

**Comment by euu2021** [comment]
Nice!

I think that videos explaining the Freeplane codebase would be valuable.
